### ISSUE TITLE
Add root support

### ISSUE THREAD

**Issue by ghost** [MuntashirAkon/CaptivePortalController#7]
Add support for automatically granting WRITE_SECURE_SETTINGS permission via root access. After a quick search on stackoverflow it seems to be possible to do, but sorry if I am wrong.

**Comment by MuntashirAkon** [comment]
Yes, possible. You can run the following command to grant it the permission:

```sh
pm grant io.github.muntashirakon.captiveportalcontroller android.permission.WRITE_SECURE_SETTINGS 
```

**Comment by nutpantz** [comment]
[CPC_Write_Secure1.zip](https://github.com/MuntashirAkon/CaptivePortalController/files/15070384/CPC_Write_Secure1.zip)

could you not make a flash able zip to run the command in magisk also???
( Warning this is not something i think is safe to use for anything but a doorstop. it has never been tested and should never be used, unless someone who knows something about magisk can look into it.)
( so NEVER FLASH IT)
(Not that it even works)
